### PR TITLE
Update usaco-762.mdx

### PR DESCRIPTION
**Pull request by Sanjana-Reddy-Kakulavaram**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]
indent with tabs

File: solutions/usaco-762.mdx [hunk 1]
```diff
@@ -11,7 +11,61 @@ author: Juheon Rhee
...
+<CPPSection>
+
+```cpp
+#include<bits/stdc++.h>
+using namespace std;
+typedef long long ll;
+
+int main() {
+        freopen("homework.in", "r", stdin);
+        ll n;
```
Comment (envyaims) [inline solutions/usaco-762.mdx:24]: avoid making everything `ll`

File: solutions/usaco-762.mdx [hunk 2]
```diff
@@ -11,7 +11,61 @@ author: Juheon Rhee
...
+        ll right = n;
+
+        vector<ll> total(n, 0);
+        vector<ll> minimum(n, 0);
+        vector<ll> max_score_values;
+
+        total[n - 1] = questions[n - 1];
+        minimum[n - 1] = questions[n - 1];
+
+        ll cur_q, cur_s, idx;
```
Comment (envyaims) [inline solutions/usaco-762.mdx:40]: expand these variable names

File: solutions/usaco-762.mdx [hunk 3]
```diff
@@ -11,7 +11,61 @@ author: Juheon Rhee
...
+        cin >> n;
+        vector<ll> questions(n);
+        for (ll i = 0; i < n; i++) {
+                cin >> questions[i];
+        }
+        ll left = -1;
+        ll right = n;
+
+        vector<ll> total(n, 0);
+        vector<ll> minimum(n, 0);
```
Comment (envyaims) [inline solutions/usaco-762.mdx:34]:
```suggestion
        vector<ll> minimum(n);
```
unnecessary, same for others

File: solutions/usaco-762.mdx [hunk 4]
```diff
@@ -11,7 +11,61 @@ author: Juheon Rhee
...
+
+        ll cur_q, cur_s, idx;
+        for (ll i = n - 2; i > 0; i--) {
+                cur_q = questions[i];
+
+                total[i] = cur_q + total[i + 1];
+                minimum[i] = min(minimum[i + 1], cur_q);
+
+                cur_s = total[i] - minimum[i];
+                idx = (n - (i + 1));
```
Comment (envyaims) [inline solutions/usaco-762.mdx:48]: some comments explaining what you are exactly doing here would be nice

**Review by envyaims — CHANGES_REQUESTED** [review]

File: solutions/usaco-762.mdx (hunk 4 above)
Comment (Sanjana-Reddy-Kakulavaram) [reply solutions/usaco-762.mdx:48]: I have made all the changes.

File: solutions/usaco-762.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/usaco-762.mdx:40]: unaddressed

Comment (Sanjana-Reddy-Kakulavaram) [reply solutions/usaco-762.mdx:40]: I have made the changes, but the deployment was failed sorry, I have again committed the changes.

Comment (SansPapyrus683) [reply solutions/usaco-762.mdx:40]: don't think it's addressed still?

**Comment by Sanjana-Reddy-Kakulavaram** [comment]
[image: image.png]

This is the screenshot of my latest commit, when I am trying to commit
again it is not accepted as there are no changes, but I am really not able
to understand why these changes are not reflecting.

**Comment by SansPapyrus683** [comment]
sorry, i can't see the image

**Comment by Sanjana-Reddy-Kakulavaram** [comment]
vector<ll> questions(n);
        for (int i = 0; i < n; i++) {
                cin >> questions[i];
        }
        int left = -1;
        int right = n;
        vector<ll> total_score(n);
        vector<ll> minimum_score(n);
        vector<ll> max_score_values;
        total_score[n - 1] = questions[n - 1];
        minimum_score[n - 1] = questions[n - 1];
        ll cur_q, cur_s, idx;

This is the code that I wanted to share the screenshot of.

**Comment by SansPapyrus683** [comment]
yeah, the variables still aren't expanded

File: solutions/usaco-762.mdx [hunk 5]
```diff
@@ -11,7 +11,61 @@ author: Juheon Rhee
...
+        int right = n;
+
+        vector<ll> total_score_from_i_till_end(n);
+        vector<ll> minimum_score_from_i_till_end(n);
+        vector<ll> max_score_values;
+
+        total_score_from_i_till_end[n - 1] = questions[n - 1];
+        minimum_score_from_i_till_end[n - 1] = questions[n - 1];
+
+        ll cur_q, cur_s, idx;
```
Comment (envyaims) [inline solutions/usaco-762.mdx:40]: ... he didnt mean the vector names.. You forgot `cur_q` and `cur_s`.

revert the vector names bruh

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Comment by Aadit-Ambadkar** [comment]
* meant to add when everything else is changed this looks good

**Pull request merged**